tests/utils/datavolume.go

```diff
@@ -374,7 +374,7 @@ func NewDataVolumeWithHTTPImportToBlockPV(dataVolumeName string, size string, ht
 					URL: httpURL,
 				},
 			},
-			PVC: &k8sv1.PersistentVolumeClaimSpec{
+			Storage: &cdiv1.StorageSpec{
```

Comment:
This is quite a challenge to all of the tests that use this API (could even hurt their integrity, if they don't intend to test that)
I'd prefer if we don't do it in this PR at least

Reply:
Well part of the issue is the current state is also not working reliably. And it was constantly failing on this PR.

Reply:
In that case could just change the spec for those tests, but IIUC the main issue was us mutating the pointer

Reply:
There are 2 issues. 1. The size was not large enough causing a 100% failure rate. Once I made it a little larger, there was the mutating the pointer issue that got fixed with the deepcopy.